docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  db:
+    restart: always
+    image: postgres:10.3
+    environment:
+      - POSTGRES_USER=govoteuser
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=GoVoteGSO
+    volumes:
+      - ./postgres-data/postgres:/var/lib/postgresql/data
```

Comment:
Is this to persist the data should the container be deleted? Is this something we want to do?